packages/orchestrator-ui-components/src/rtk/endpoints/formFields.ts

```diff
@@ -0,0 +1,60 @@
+import { ImsNode } from '@/components';
+import { ContactPerson } from '@/components/WfoForms/formFields/types';
+import { BaseQueryTypes, orchestratorApi } from '@/rtk';
+
+const LOCATION_CODES_ENDPOINT = 'surf/crm/location_codes';
+const CONTACT_PERSONS_ENDPOINT = 'surf/crm/contacts';
+const IMS_NODES_ENDPOINT = '/surf/ims/nodes';
```

Comment:
The endpoint seems to be surf specific, consider moving them to the surf specific app

Reply:
I think we can only do that when the surf form fields are also moved